server/routes/webmanifest.js

```diff
@@ -2,17 +2,17 @@ const assets = require('../../common/assets');
 
 module.exports = function(req, res) {
   const manifest = {
-    name: 'Firefox Send',
-    short_name: 'Send',
+    name: 'Checkr Sendr',
+    short_name: 'Sendr',
     lang: req.language,
     icons: [
       {
-        src: assets.get('android-chrome-192x192.png'),
+        src: assets.get('checkr-favicon-150x150.png'),
         type: 'image/png',
-        sizes: '192x192'
+        sizes: '150x150'
       },
       {
-        src: assets.get('android-chrome-512x512.png'),
+        src: assets.get('checkr_logo_white.png'),
```

Comment:
this looks like the placeholder for the LARGE logo, if needed. This logo is the Large white Checkr logo. I dont think we have a large favicon do we? 
Also, the small icon is the 'C' (for checkr) favicon.

Are we not going with any branding for the name of Sendr? Do we intend for people to call this app by its name? or just document exchange app?

Reply:
based on the feedback from marketing, we should use `Sendr` sparingly. maybe `Checkr | Doc Exchange`. 

as for the logo/favi, check in the marketing drive

Reply:
Marketing drive: https://drive.google.com/drive/u/0/folders/1W7I5CAaiH-9IQq9m4NMGAw7Grg1Y30pA

I think the favicon must have come from the marketing design guidelines. Anyone have that link? 

The title in the app is `    title: 'Sendr | Checkr',` 
To be consistent we should also change the name in the web manifest to be the same:
`    name: 'Sendr | Checkr',`

If we should not be using Sendr, what should we call this? `Doc Exchange`  seems viable. Though we should probably rename the title too, for consistency